Makefile

```diff
@@ -0,0 +1,14 @@
+TMPDIR := $(shell mktemp -d)
+OUTDIR := $(shell pwd)
+
+_default:
+	@mkdir -p $(TMPDIR)/usr/share/anaconda/addons/
+	@cp -par org_fedora_hello_world $(TMPDIR)/usr/share/anaconda/addons/
+	
```

Comment:
Please remove these leading whitespace characters.

Reply:
I'm not sure I understand this request.  If I remove the tabs, the Makefile syntax is incorrect.

Reply:
I mean remove tabs (indentation) on the empty lines. That won't break Makefile it should work correctly.